db/schema.yml

```diff
@@ -899,8 +899,8 @@ container_templates:
 - objects
 - created_at
 - updated_at
-- object_labels
 - type
+- object_labels
```

Comment:
This should not happen.

Reply:
Maybe the migrations were run out of order?